src/modules/visits/services/visits.service.ts

```diff
@@ -63,11 +63,21 @@ export class VisitsService {
 		return true;
 	}
 
+	public static adaptSpaceAddress(graphQlVisit: any): string {
+		const path = 'space.schema_maintainer.information[0].primary_site.address';
+		const locality = get(graphQlVisit, `${path}.locality`);
+		const postalCode = get(graphQlVisit, `${path}.postal_code`);
+		const street = get(graphQlVisit, `${path}.street`);
+
+		return `${street}, ${postalCode} ${locality}`;
+	}
+
 	public adapt(graphQlVisit: any): Visit {
 		return {
 			id: get(graphQlVisit, 'id'),
 			spaceId: get(graphQlVisit, 'cp_space_id'),
 			spaceName: get(graphQlVisit, 'space.schema_maintainer.schema_name'),
+			spaceAddress: VisitsService.adaptSpaceAddress(graphQlVisit),
```

Comment:
alternative so that the adaptSpaceAddress does not need to know where the address is stored in the parent object. It only needs to adapt the address that it receives.
```suggestion
	public static adaptSpaceAddress(graphQlSpaceAddress: any): string {
		const locality = get(graphQlSpaceAddress, 'locality');
		const postalCode = get(graphQlSpaceAddress, 'postal_code');
		const street = get(graphQlSpaceAddress, 'street');

		return `${street}, ${postalCode} ${locality}`;
	}

	public adapt(graphQlVisit: any): Visit {
		return {
			id: get(graphQlVisit, 'id'),
			spaceId: get(graphQlVisit, 'cp_space_id'),
			spaceName: get(graphQlVisit, 'space.schema_maintainer.schema_name'),
			spaceAddress: VisitsService.adaptSpaceAddress(get(graphQlVisit, 'space.schema_maintainer.information[0].primary_site.address')),
```